security_standards.md

```diff
@@ -0,0 +1,114 @@
+
+
+
+
+
+# Security Standard.
+
+# Background
+
+Having a secure approach to development has never been so important.
+
+The way we build software and systems is rapidly evolving, becoming more and more automated and integrated. This results in a need to have some standards and guidance around security. To save our time looking up and picking ways and more importantly to keep things uniform so things are done in the best way across the board
+
+This guidance will help you understand the security implications of modern code development and deployment practices.
+
+
+
+## Principles
```

Comment:
I would suggest moving this section on principles to its own document. Then if PR #9 gets approved it could live in the principles folder.